src/js/components/Drop/DropContainer.js

```diff
@@ -101,7 +96,7 @@ const DropContainer = forwardRef(
 
     useEffect(() => {
       const notifyAlign = () => {
-        const styleCurrent = (ref || dropRef).current.style;
+        const styleCurrent = dropRef?.current.style;
```

Comment:
```suggestion
        const styleCurrent = dropRef.current.style;
```

Reply:
We're going to dereference `styleCurrent` anyway on the next line, so returning undefined won't help.